src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,33 @@
+package duke;
+
+import java.time.LocalDateTime;
+
+public class Deadline extends Task {
+
+    /**
+     * Constructor to initialise the task.
+     * @param task The task itself.
+     * @param type The type of the task defined by the enum duke.Type.
+     * @param done The status of the task.
+     * @param datetime The time for task.
+     */
+    public Deadline(String task, Type type, Boolean done, LocalDateTime datetime) {
+        super(task, type, done, datetime);
+    }
+
+    /**
+     * String reflecting type of task.
+     * @return String reflecting type of task.
+     */
+    public String typeString() {
+        return "[D]";
+    }
+
+    /**
+     * String reflecting task itself.
```

Comment:
I think "String reflecting this task" would fit the JavaDocs conventions better?